### PR TITLE
Adding 5 new sample apps, adding Monitor SDKs, updating Android SDK to the current/new one.

### PR DESCRIPTION
**Pull request by InbarGazit**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -226,9 +226,63 @@
...
+        "node_id": "MDEwOlJlcG9zaXRvcnkzNjg2Njk5OTk=",
+        "github_name": "docusign-monitor-csharp-client",
+        "github_url": "https://github.com/docusign/docusign-monitor-csharp-client",
+        "display_name": "DocuSign C# Monitor SDK",
+        "description": "The official DocuSign Monitor C# SDK (Client Library) used to interact with the DocuSign Monitor API.",
+        "language": "c#",
+        "tags": [ "opensource", "csharp", "monitor", "client-library", "sdk" ]
+      },
+      {
+        "node_id": "MDEwOlJlcG9zaXRvcnkzNjcxODE5MTE",
```
Comment (annesophien) [inline manifest.json:261]: node_id is missing an equal sign at the end. Should be `MDEwOlJlcG9zaXRvcnkzNjcxODE5MTE=`

File: manifest.json [hunk 2]
```diff
@@ -804,6 +858,51 @@
...
+        "tags": [ "opensource", "java", "android", "esignature", "sample-app" ]
+      },
+      {
+        "node_id": "R_kgDOGQwYSg",
+        "github_name": "sample-app-tgk-financial-ios",
+        "github_url": "https://github.com/docusign/sample-app-tgk-financial-ios",
+        "display_name": "TGK Financial iOS Sample App",
+        "description": "TGK Financial sample application showcases how a fictitious financial institution can use the DocuSign iOS SDK to build a mobile application for their customers.",
+        "language": "java",
+        "tags": [ "opensource", "objc", "ios", "esignature", "sample-app" ]
```
Comment (annesophien) [inline manifest.json:886]: The language and tags don't match up

File: manifest.json [hunk 3]
```diff
@@ -804,6 +858,51 @@
...
+        "node_id": "R_kgDOGemeMQ",
+        "github_name": "sample-app-myfinance-apex",
+        "github_url": "https://github.com/docusign/sample-app-myfinance-apex",
+        "display_name": "MyFinance Sample App",
+        "description": "MyFinance is a DocuSign sample application showcasing how you can integrate DocuSign and Salesforce with the Apex Toolkit.",
+        "language": "apex",
+        "tags": [ "opensource", "apex", "salesforce", "esignature", "sample-app" ]
+      },
+      {
+        "node_id": "R_kgDOGQwYSg",
```
Comment (annesophien) [inline manifest.json:898]: The correct node_id is `R_kgDOGhvnAg`

File: manifest.json [hunk 4]
```diff
@@ -34,37 +34,37 @@
     "SDKs": [
       {
         "node_id": "MDEwOlJlcG9zaXRvcnkxMjkzOTAzOQ==",
-        "github_name": "docusign-java-client",
+        "github_name": "docusign-esign-java-client",
         "github_url": "https://github.com/docusign/docusign-java-client",
```
Comment (annesophien) [inline manifest.json:38]: Please update to new GH URL: https://github.com/docusign/docusign-esign-java-client

File: manifest.json [hunk 5]
```diff
@@ -34,37 +34,37 @@
...
+        "display_name": "DocuSign eSignature Java SDK",
+        "description": "The official DocuSign eSignature Java SDK (Client Library) used to interact with the DocuSign eSignature REST API. Send, sign, and approve documents using this client",
         "language": "java",
         "tags": [ "opensource", "java", "esignature", "client-library", "sdk" ]
       },
       {
         "node_id": "MDEwOlJlcG9zaXRvcnkxMzQwMzI0OA==",
-        "github_name": "docusign-php-client",
+        "github_name": "docusign-esign-php-client",
         "github_url": "https://github.com/docusign/docusign-php-client",
```
Comment (annesophien) [inline manifest.json:47]: https://github.com/docusign/docusign-esign-php-client

File: manifest.json [hunk 6]
```diff
@@ -79,27 +79,27 @@
...
+        "github_url": "https://github.com/docusign/docusign-esign-python-client",
+        "display_name": "DocuSign eSignature Python Client",
+        "description": "The official DocuSign eSignature Python SDK (Client Library) used to interact with the DocuSign eSignature REST API. Send, sign, and approve documents using this client. https://www.docusign.com/devcenter",
         "language": "python",
         "tags": [ "opensource", "python", "esignature", "client-library", "sdk" ]
       },
       {
         "node_id": "MDEwOlJlcG9zaXRvcnkxMDM0MzEyMDk=",
         "github_name": "docusign-ruby-client",
         "github_url": "https://github.com/docusign/docusign-ruby-client",
```
Comment (annesophien) [inline manifest.json:92]: https://github.com/docusign/docusign-esign-ruby-client

**Review by annesophien — CHANGES_REQUESTED** [review]
Thanks for making the updates! I left some comments to be addressed

File: manifest.json (hunk 1 above)
Comment (InbarGazit) [reply manifest.json:261]: done

File: manifest.json (hunk 2 above)
Comment (InbarGazit) [reply manifest.json:886]: done

File: manifest.json (hunk 3 above)
Comment (InbarGazit) [reply manifest.json:898]: done

File: manifest.json (hunk 4 above)
Comment (InbarGazit) [reply manifest.json:38]: done

File: manifest.json (hunk 6 above)
Comment (InbarGazit) [reply manifest.json:92]: done

**Review by annesophien — APPROVED** [review]

**Pull request merged**